README.md

```diff
@@ -24,6 +24,12 @@ there are few/no changes, but is able to output each differing row! By pushing
 the compute into the databases, it's _much_ faster than querying for and
 comparing every row.
 
+![Performance for 100M rows](https://user-images.githubusercontent.com/97400/174860361-35158d2b-0cad-4089-be66-8bf467058387.png)
+
+**†:** Downloading all row implementation is not optimal. It's a single-process,
```

Comment:
This sentence needs a bit of work.

Reply:
Can you tell me what you think needs work about it?

Reply:
Maybe just rephrase it:

The Python implementation for downloading rows isn't optimal in some of the drivers. It's a single process....

Reply:
There's more to it than that, because of the GIL, etc. I did another pass and will merge